protocol/src/types/executor.rs

```diff
@@ -52,6 +52,7 @@ pub struct ExecutorContext {
     pub gas_price:              U256,
     pub block_gas_limit:        U256,
     pub block_base_fee_per_gas: U256,
+    pub extra_data:             bytes::Bytes,
```

Comment:
`use crate::types::Bytes`
```suggestion
    pub extra_data:             Bytes,
```